#parameters are in [P, I, D] format
angle_PID: [0.9, 0.0, 0.6]
dist_PID: [0.25, 0.0, 0.8]
oscill_thresh: 0.01
target_offset: 1.26
radius: 5.50
goal_dist_thresh: 0.5
y_samples: 5
